src/west/_bootstrap/main.py

```diff
@@ -247,6 +254,15 @@ def bootstrap(args):
     with open(os.path.join(directory, WEST_DIR, WEST_MARKER), 'w') as f:
         hide_file(f.name)
 
+    # Wrap west init with --cached:
```

Comment:
I think the init command should remain the responsibility of the bootstrapper. This behavior is confusing to me and feels a bit brittle, in my opinion. Each bootstrapper should be in charge of its own init procedure. Not to mention making printing help a bit weird...

Reply:
I am confused, didn't we discuss this recently and agreed that we would instead defer to the main installation after cloning the manifest repository in a temporary location?

Reply:
Dropping this concern based on discussion elsewhere. Never mind :)